tests/drivers/spi/spi_loopback/testcase.yaml

```diff
@@ -201,3 +201,6 @@ tests:
     platform_allow:
       - frdm_ke17z
       - frdm_ke17z512
+  drivers.spi.max32_dma.loopback:
+    extra_args: OVERLAY_CONFIG="overlay-max32-spi-dma.conf"
+    filter: CONFIG_SOC_FAMILY_MAX32
```

Comment:
Why do you need to exclude dma configuration?
This line will enable CONFIG_SPI_MAX32_DMA flag while it be executed by twister, but during manual testing DMA will not be enabled unless it specified.

Reply:
Interrupt mode won't be tested if we leave DMA enabled. We cannot test them both at the same time.